src/Cache_Command.php

```diff
@@ -339,4 +339,36 @@ public function type( $args, $assoc_args ) {
 		WP_CLI::line( $message );
 	}
 
+	/**
+	 * Removes all cache items in a group, if the object cache implementation supports it.
+	 *
+	 * ## OPTIONS
+	 *
+	 * <group>
+	 * : Cache group key.
+	 *
+	 * ## EXAMPLES
+	 *
+	 *     # Clear cache group.
+	 *     $ wp cache clear-group my_group
+	 *
+	 * @subcommand clear-group
+	 */
+	public function clear_group( $args, $assoc_args ) {
+		list( $group ) = $args;
+
+		if ( ! function_exists( 'wp_cache_supports' ) ) {
+			WP_CLI::error( 'Checking cache features is only available in WordPress 6.1 and higher' );
+		}
```

Comment:
Can we consolidate this conditional with the `if ( ! wp_cache_supports( 'flush_group' ) ) {` conditional below?

I think it's fine for both of them to say 'Group flushing is not supported.'